modules/stats.php

```diff
@@ -573,42 +758,48 @@ function stats_configuration_head() {
 	<?php
 }
 
+/**
+ * Stats Configuration Screen.
+ *
+ * @access public
+ * @return void
+ */
 function stats_configuration_screen() {
 	$options = stats_get_options();
-	?>
+?>
 	<div class="narrow">
-		<p><?php printf( __( 'Visit <a href="%s">Site Stats</a> to see your stats.', 'jetpack' ), esc_url( menu_page_url( 'stats', false ) ) ); ?></p>
+		<p><?php printf( esc_attr_e( 'Visit <a href="%s">Site Stats</a> to see your stats.', 'jetpack' ), esc_url( menu_page_url( 'stats', false ) ) ); ?></p>
```

Comment:
`esc_attr_e` will output the string without placeholder values inserted. Plus, it's the wrong context, let's use `esc_html__` instead.